src/main/java/com/epam/aidial/core/service/LockService.java

```diff
@@ -0,0 +1,95 @@
+package com.epam.aidial.core.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.redisson.api.RScript;
+import org.redisson.api.RedissonClient;
+import org.redisson.client.codec.StringCodec;
+
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.LockSupport;
+import javax.annotation.Nullable;
+
+@Slf4j
+public class LockService {
```

Comment:
Can you please add one line comment, why we did LockService (that it expected to work with `volatile-lfu`

Reply:
Added.